locale/templates/settings.ftl

```diff
@@ -535,7 +535,7 @@ se-content-note = Note: a secondary email won’t restore your information — y
 tfa-row-header = Two-step authentication
 tfa-row-disabled = Two-step authentication disabled.
 tfa-row-enabled = Enabled
-tfa-row-not-set = Not set
+tfa-row-not-set = Not Set
```

Comment:
If the change is a must, then we need a new string ID otherwise, reusing the same ID will not trigger Pontoon as a new or revised string - this is the case for those locales already translated the strings, including en-CA and en-GB.